### PR TITLE
Migrate Emoji.Search to Sync by default

### PR DESCRIPTION
**Pull request by gupsho**
#### Summary
Migrate "Emoji.Search" to Sync by default 

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/11292

### CODE REVIEW

File: store/sqlstore/emoji_store.go [hunk 1]
```diff
@@ -183,19 +183,18 @@ func (es SqlEmojiStore) Delete(id string, time int64) store.StoreChannel {
...
+	}
 
-		term += name + "%"
+	term += name + "%"
 
-		if _, err := es.GetReplica().Select(&emojis,
-			`SELECT
+	if _, err := es.GetReplica().Select(&emojis,
+		`SELECT
 				*
```
Comment (jespino) [inline store/sqlstore/emoji_store.go]: Can you remove 1 tab indentation here to the rest of the query to maintain it consistent with the previous existing indentation?

Comment (gupsho) [reply store/sqlstore/emoji_store.go]: Done.

**Comment by gupsho** [comment]
> Just an small indentation thing, but otherwise looks good to me. Thanks @gupsho!

Done.

**Comment by jespino** [comment]
@gupsho @saturninoabril done :)

**Pull request merged**